### PR TITLE
p_tree updates for admin builder.

### PR DESCRIPTION
**Pull request by gknisely**
updated logic to match the updated config

### CODE REVIEW

File: src/mjolnir/pbfadminbuilder.cc [hunk 1]
```diff
@@ -242,13 +242,15 @@ void BuildAdminFromPBF(const boost::property_tree::ptree& pt,
   // relations are defined within the PBFParser class
   OSMData osmdata = PBFAdminParser::Parse(pt, input_files);
 
-  std::string dir = pt.get<std::string>("admin.admin_dir");
-  std::string db_name = pt.get<std::string>("admin.db_name");
-
-  std::string database = dir + "/" +  db_name;
+  // Bail if bad path
+  auto database = pt.get_optional<std::string>("admin");
+  if((*database).empty() || !boost::filesystem::exists(boost::filesystem::path(*database).parent_path()) ) {
```
Comment (kevinkreiser) [inline src/mjolnir/pbfadminbuilder.cc]: this should be `!database || ...`

Comment (kevinkreiser) [inline src/mjolnir/pbfadminbuilder.cc]: dont need to check parent path exists, just need to make_dirs the parent path

**Comment by gknisely** [comment]
@kevinkreiser updated

**Comment by kevinkreiser** [comment]
looks good to me

**Pull request merged**